src/components/util/validation-functions.js

```diff
@@ -48,7 +48,7 @@ export function isBlankOrEmpty(value) {
  * Returns true if the value is a valid number, per Gridsuite's standard (allows either coma or dots for decimal).
  */
 export function validateValueIsANumber(value) {
-    if (value === undefined || value === '') {
+    if (value === null || value === '') {
```

Comment:
I think the remark was to use "if (value **==** null || value === '')". Here this condition would pass if value is equal to "undefined"
I am not a fan of implicit coercion though